lib/chef/resource/windows_certificate.rb

```diff
@@ -58,7 +58,21 @@ class WindowsCertificate < Chef::Resource
 
       action :create do
         description "Creates or updates a certificate."
-        add_cert(OpenSSL::X509::Certificate.new(raw_source))
+
+        cert_obj = OpenSSL::X509::Certificate.new(raw_source) # A certificate object in memory
```

Comment:
can we get a comment on the raw_source method explaining what's it's doing

Reply:
Yes, I will do that. Thanks

Reply:
Have added the comment. Thanks